spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -132,6 +132,17 @@ def associate_groups_with_app(app, groups)
       details_for_app(app)
     end
 
+    def valid_name_for(input)
+      latinazed = input.to_slug.transliterate.to_s # remove accents
+      latinazed = latinazed.gsub(/[^0-9A-Za-z\d\s]/, '') # remove non-valid characters
+      # it may result in totaly empty strings that only contain chinese symbols.
+      if latinazed != input
```

Comment:
the thing is, when we start `gsub` it may result in a app_name that already exists.
and it would be hard to explain to the users to choose another name, without a non-ascii char.
therefore i thought that checking if it has been modified, and if so, append md5